apps/web/src/components/PoolCandidatesTable/helpers.tsx

```diff
@@ -0,0 +1,360 @@
+import React from "react";
+import { IntlShape } from "react-intl";
+import { SortingState } from "@tanstack/react-table";
+
+import {
+  commonMessages,
+  getCandidateSuspendedFilterStatus,
+  getLanguage,
+  getPoolCandidatePriorities,
+  getPoolCandidateStatus,
+  getProvinceOrTerritory,
+} from "@gc-digital-talent/i18n";
+import { parseDateTimeUtc } from "@gc-digital-talent/date-helpers";
+import { Spoiler } from "@gc-digital-talent/ui";
+import {
+  OrderByRelationWithColumnAggregateFunction,
+  PoolCandidateSearchInput,
+  QueryPoolCandidatesPaginatedOrderByRelationOrderByClause,
+  QueryPoolCandidatesPaginatedOrderByUserColumn,
+} from "@gc-digital-talent/graphql";
+
+import {
+  CandidateSuspendedFilter,
+  Language,
+  PoolCandidate,
+  PoolCandidateStatus,
+  ProvinceOrTerritory,
+  SortOrder,
+  User,
+} from "~/api/generated";
+import useRoutes from "~/hooks/useRoutes";
+import { getFullNameLabel } from "~/utils/nameUtils";
+
+import cells from "../Table/cells";
+
+export const statusAccessorNew = (
+  status: PoolCandidateStatus | null | undefined,
+  intl: IntlShape,
+) => {
+  if (status === PoolCandidateStatus.NewApplication) {
+    return (
+      <span
+        data-h2-color="base(tertiary.darker)"
+        data-h2-font-weight="base(700)"
+      >
+        {status
+          ? intl.formatMessage(getPoolCandidateStatus(status as string))
+          : ""}
```

Comment:
Not sure if this is in scope. But, we check to see if status exists 3 times in this function. I think it would be an easy win if instead we just check that at the start and return `null` instead. Also, we won't have an empty span tag as well.

```tsx
if(!status) return null;

// Now we know it exists and can render different messages based on it
```

Reply:
Fixed by [95daca8](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8360/commits/95daca8101e574241aa2f72decff7b9cc7c12cdd)